client/src/pages/addNewSitSpot/index.js

```diff
@@ -0,0 +1,87 @@
+import React, { Component } from 'react';
+import axios from 'axios';
+import { StepsQuestions } from '../../components';
+
+import Questions from './fakeData';
+
+import './index.css';
+
+const Classes = [
+  'name',
+  'website',
+  'country',
+  'city',
+  'type',
+  'upload-img',
+  'upload-img',
+];
+
+class AddNewSitSpot extends Component {
+  state = {
+    data: {
+      name: '',
+      linkSite: '',
+      country: '',
+      city: '',
+      businessType: '',
+      img1: undefined,
+      img2: undefined,
+    },
+    currentStep: 0,
+  };
+
+  onSubmit = () => {
+    const { data } = this.state;
+    const formData = new FormData();
+    Object.entries(data).forEach(
+      ([key, value]) => !!value && formData.append(key, value)
+    );
+
+    axios.post('/api/v1/place', formData, {
+      headers: { 'content-type': 'multipart/formdata' },
+    });
+    // .then(result => console.log(result));
+  };
+
+  handleChange = (value, dataKey) => {
+    this.setState(prevState => ({
+      ...prevState,
+      data: { ...prevState.data, [dataKey]: value },
+    }));
+  };
+
+  next = () =>
```

Comment:
Dont let the user go to the next before doing the current step!

Reply:
good point but we ill leave it to asem.. im scared to touch his component code